src/NumericalAlgorithms/SphericalHarmonics/SpherepackCache.cpp

```diff
@@ -0,0 +1,17 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "NumericalAlgorithms/SphericalHarmonics/SpherepackCache.hpp"
+
+#include <cstddef>
+
+#include "NumericalAlgorithms/SphericalHarmonics/Spherepack.hpp"
+#include "Utilities/StaticCache.hpp"
+
+namespace ylm {
+const Spherepack& get_spherepack_cache(const size_t l_max) {
+  static const auto spherepack_cache = make_static_cache<CacheRange<2, 21>>(
```

Comment:
I think SpEC does up to l=150... Is compile time terrible if we set it to that? Or maybe l=100?